dashboard/app/models/user.rb

```diff
@@ -1332,6 +1332,13 @@ def authorized_teacher?
     false
   end
 
+  def authorized_instructor?
```

Comment:
Do we expect `authorized_instructor?` to eventually replace `authorized_teacher?` (and delete `authorized_teacher?`)?

Also, FWIW, I find "verified" to be clearer than "authorized" (but can go with other given our historical usage of authorized).

Reply:
The plan is for `authorized_instructor?` to replace `authorized_teacher?` as we do the work to move fully onto the new instructor and participant audiences. 

I don't have a strong opinion on verified vs authorized. We seem to use both. The permission for teachers is called AUTHORIZED_TEACHER but we often say verified teacher. I can update it to verified if that feels better. @davidsbailey  thoughts?

Reply:
I have updated to verified but I can always go back.

Reply:
I'm fine with either. we use these pretty interchangeably, so I think we'll be stuck with these as synonyms for some time.